CHANGELOG.md

```diff
@@ -35,6 +35,35 @@ RELEASE CHANGELOG TEMPLATE:
 ### Documentation
 -->
 
+<a name="v2.0.1"></a>
+# [2.0.1](https://github.com/ligato/vpp-agent/compare/v2.0.0...master) (2019-04-05)
```

Comment:
The comparison is done between two consecutive releases. That way it will list same changes no matter what happens to master. 

In this case it should be: `v2.0.0...v2.0.1`.